lib/handler/configurator/proxy.c

```diff
@@ -618,10 +618,21 @@ static int on_config_http3_ratio(h2o_configurator_command_t *cmd, h2o_configurat
 static int on_config_expect(h2o_configurator_command_t *cmd, h2o_configurator_context_t *ctx, yoml_t *node)
 {
     struct proxy_configurator_t *self = (void *)cmd->configurator;
-    ssize_t ret = h2o_configurator_get_one_of(cmd, node, "OFF,ON");
-    if (ret == -1)
+    ssize_t ret = h2o_configurator_get_one_of(cmd, node, "OFF,ON,FORWARD");
+    switch (ret) {
+    case 0:
+        self->vars->conf.expect_mode = H2O_PROXY_EXPECT_DISABLED;
+        break;
+    case 1:
+        self->vars->conf.expect_mode = H2O_PROXY_EXPECT_ENABLED;
```

Comment:
Sorry to suggest this late, but would it make sense to call these ~_OFF and ~_ON, considering that they are referred to as such by the config yaml?

Reply:
Renamed in https://github.com/h2o/h2o/pull/3419/commits/faf6912fc323f6f546d18b56afba5f247fa7b9f4